update_unix.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+// RunUpdate runs the update on the current tgf executable
+func RunUpdate() bool {
+	executablePath, err := os.Executable()
+	if err != nil {
+		printWarning("Error getting executable path", err)
+	}
+
+	currentDir := filepath.Dir(executablePath)
+
+	os.Setenv("TGF_PATH", currentDir)
+	os.Setenv("TGF_LOCAL_VERSION", version)
+
+	updateScript, fetchErr := fetchUpdateScript()
+	if fetchErr != nil {
+		printWarning("Error fetching update script: ", fetchErr)
+	}
+
+	output, errc := exec.Command("bash", "-c", updateScript).Output()
+
+	if errc != nil {
+		log.Fatal("Error running update script: ", errc.Error())
+		log.Fatal(string(output))
```

Comment:
log.Fatal completely exits the process. This line is unreachable

Reply:
Also, an update error should absolutely not exit the process. It should still execute the tgf process